packages/instrumenter/src/transformers/directive-bookkeeper.ts

```diff
@@ -58,6 +64,7 @@ export class DirectiveBookkeeper {
       )
       .filter(notEmpty)
       .forEach(([, directiveType, scope, mutators, optionalReason]) => {
+        this.warnAboutUnusedDirective(mutators, directiveType, scope!);
         const mutatorNames = mutators.split(',').map((mutator) => mutator.trim().toLowerCase());
```

Comment:
Please pass this `mutatorNames` to `warnAboutUnusedDirecttive` instead of `mutators`. That way we only split on `,` once.

Reply:
Is it OK to show lowercase mutator names, in that case I don't need to split op the assignment to the `mutatorNames`?